webui/react/src/pages/ExperimentList.tsx

```diff
@@ -228,6 +233,28 @@ const ExperimentList: React.FC<Props> = ({ project }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [id, isLoadingSettings, settings, labelsString, pinnedString, statesString, usersString]);
 
+  const fetchHiddenSelectedExperiments = useCallback(async (): Promise<void> => {
+    try {
+      if ((settings.row?.length ?? 0) > 0) {
```

Comment:
`if (settings.row?.length) {` ?

Reply:
using `hiddenExpIds.length > 0` here, so no need to handle nullish aymore